sympy/functions/elementary/tests/test_miscellaneous.py

```diff
@@ -410,6 +410,10 @@ def test_issue_12638():
     assert Min(a, b, Max(a, b, c)) == Min(a, b)
     assert Min(a, b, Max(a, c)) == Min(a, b)
 
+def test_issue_21399():
+    from sympy.abc import a, b, c
+    assert Max( Min(a, b), Min(a, b, c)) == Min(a,b)
```

Comment:
The spacing here is off. There should be no space after `(` but there should be a space after `,` so:
```
assert Max(Min(a, b), Min(a, b, c)) == Min(a, b)
```

Reply:
Also the line spacing between test functions here should be two blank lines between each function. Can you fix that for this function and also those below?